app/main.py

```diff
@@ -1,3 +1,18 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        import math
+        return math.ceil(days / 7)
```

Comment:
Import units inside method or function - it's a bad practice.

Reply:
Please, change status in exsel if you check another code